src/braintree-manager/payment-providers/paypal/paypal-button-datasource.ts

```diff
@@ -182,22 +180,19 @@ export class PayPalButtonDataSource implements PayPalButtonDataSourceInterface {
   async onAuthorize(data: paypal.AuthorizationData): Promise<paypal.TokenizePayload> {
     const payload: paypal.TokenizePayload = await this.paypalInstance.tokenizePayment(data);
 
-    console.debug('onAuthorize data', data, 'payload', payload);
-
     this.delegate?.payPalPaymentAuthorized(this, payload);
 
     return payload;
   }
 
   /** @inheritdoc */
   onCancel(data: object): void {
-    console.debug('cancel', data);
     this.delegate?.payPalPaymentCancelled(this, data);
   }
 
   /** @inheritdoc */
   onError(error: string): void {
-    console.error('error', error);
+    console.error('PayPal error', error);
     this.delegate?.payPalPaymentError(this, error);
```

Comment:
callback to the delegate will route up any issues 👍

Reply:
Yep yep! 😄